### DISCUSSION TITLE
Error when installing Geschichte 10.0.0

### DISCUSSION THREAD

**Discussion by Morritz** [BowlingX/geschichte#37] [Q&A] [answered]
```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: geschichte@10.0.0
npm WARN Found: zustand@undefined
npm WARN node_modules/zustand
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer zustand@">=4" from geschichte@10.0.0
npm WARN node_modules/geschichte
npm WARN   geschichte@"*" from the root project
npm ERR! code ETARGET
npm ERR! notarget No matching version found for zustand@>=4.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```
Steps to reproduce:
1. npm create vite@latest
2. select react-ts
3. npm install geschichte

**Comment by BowlingX** [comment, marked as answer]
Thank you for your Question.

As `zustand` is declared as `peerDependency`, you have to add it yourself in your project.
So with `npm` that would be `npm install zustand@4.0.0-rc.1`

However the `peerDependencies` was wrongly declared in version `10.0.0`, this is now fixed with version `10.0.2` to allow `rc` versions as well.
Thank you for the Feedback!
